eng/Versions.props

```diff
@@ -69,7 +69,7 @@
       Source-build builds the product with the most recent previously source-built release. Thankfully, these two requirements line up nicely
       such that any version that satisfies the VS version requirement will also satisfy the .NET SDK version requirement because of how we ship.
     -->
-    <MicrosoftCodeAnalysisVersion_LatestVS>4.5.0</MicrosoftCodeAnalysisVersion_LatestVS>
+    <MicrosoftCodeAnalysisVersion_LatestVS>4.7.0</MicrosoftCodeAnalysisVersion_LatestVS>
```

Comment:
Would you mind cleaning up the [source build allowed prebuilt entry](https://github.com/dotnet/runtime/blob/35dd6b83dddd5803fec881d662bf7802b42bb3c1/eng/SourceBuildPrebuiltBaseline.xml#L12) for the 4.5.0 version.  I conjecture this is no longer needed.